src/pages/steps/AdditionalInformationStep/PriceInformation.tsx

```diff
@@ -441,5 +417,5 @@ const PriceInformation = ({
   );
 };
 
-export type { PriceCategory };
-export { PriceInformation };
+export type { FormData, PriceCategory };
+export { PriceCategory as PriceCategories, PriceInformation };
```

Comment:
You don't have to rename the constant per se. Having a contant and type with the same name actually has a nice DX to it. Since typescript let's you use the type as type and constant as constant, it feels like enums without the drawbacks 😉

Reply:
Does seem to throw a TS error though, do I need to use a specific syntax?

![Screenshot 2023-02-14 at 13 52 13](https://user-images.githubusercontent.com/1321596/218744460-3a090097-8ec5-4217-b5a6-92e1379999d1.png)